rocAL/include/pipeline/node.h

```diff
@@ -39,8 +39,8 @@ class Node {
     virtual ~Node();
     void create(std::shared_ptr<Graph> graph);
     void update_parameters();
-    std::vector<Tensor *> input() { return _inputs; };
-    std::vector<Tensor *> output() { return _outputs; };
+    const std::vector<Tensor *>& input() const { return _inputs; }
```

Comment:
Why did we change to const?

Reply:
Since it does not modify the inputs and just returns it, it has been changed to const